.circleci/config.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           command: |
             apk add --no-cache --no-progress py2-pip
             pip install awscli
-            ecr_login="$(aws ecr get-login --region eu-west-1 --no-include-email)"
+            ecr_login="$(aws ecr get-login --region $AWS_DEFAULT_REGION --no-include-email)"
```

Comment:
_nice to have_
🙋‍♂ `--region $AWS_DEFAULT_REGION` might be redundant and could be omitted.

Reply:
<img width="644" alt="Screen Shot 2019-04-30 at 13 04 18" src="https://user-images.githubusercontent.com/1237986/56960505-f8f2f480-6b48-11e9-9cbe-82234c9d18e0.png">
This was an oversight, but on quick inspection doesn't appear to be the case

Reply:
TIL! 🙇